presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1662,6 +1717,7 @@ VeloxQueryPlanConverter::toVeloxQueryPlan(
       node->columnNames,
       insertTableHandle,
       outputType,
+      connector::WriteProtocol::CommitStrategy::kNoCommit,
```

Comment:
Why is no-commit protocol hard-coded here?

Reply:
Here is where a system sets the commit strategy it's going to use. CommitStrategy::kNoCommit is set for Presto.

Also discussed with MJ previously, and for PrestoSpark, he will change here to add a condition to set kTaskCommit for PrestoSpark and kNoCommit for Presto.